client/src/components/SelectWidgetsModal/SelectWidgetsModal.test.js

```diff
@@ -0,0 +1,15 @@
+import React from "react";
+import SelectWidgetsModal from ".";
+import Enzyme, { ReactWrapper, shallow } from "enzyme";
+import Adapter from "enzyme-adapter-react-16";
+import Dialog from "@material-ui/core/Dialog";
+import Button from "@material-ui/core/Button";
+
+Enzyme.configure({ adapter: new Adapter() });
```

Comment:
I think this can be moved to the setupTests.js file and the project should already be configured for that file to run with `npm test` https://stackoverflow.com/questions/46627353/where-should-the-enzyme-setup-file-be-written

Reply:
![image](https://user-images.githubusercontent.com/52348242/111698960-5a86ee00-889c-11eb-8dd4-da0931909e7f.png)
When i try to update jest in package.json() I get this error - looking into it now

Reply:
Seems like without the package.json() change it still works - will update the PR now